include/deviceconfig.h

```diff
@@ -376,12 +377,7 @@ class DeviceConfig : public IJSONSerializable
 
     void SetBrightness(int newBrightness)
     {
-        if (newBrightness < BRIGHTNESS_MIN)
-            SetAndSave(brightness, BRIGHTNESS_MIN);
-        else if (newBrightness > BRIGHTNESS_MAX)
-            SetAndSave(brightness, BRIGHTNESS_MAX);
-        else
-            SetAndSave(brightness, uint8_t(newBrightness));
+        SetAndSave(brightness, std::clamp(uint8_t(newBrightness), BRIGHTNESS_MIN, BRIGHTNESS_MAX));

```

Comment:
You're right in the sense that I'm casting the wrong way around. I need to cast the MIN and MAX values up to int for the whole thing to work (they're set to uint8_t specifically in part to remind people fiddling with them that the _type_ boundaries are 0 and 255). Casting _is_ necessary because the parameters to clamp all have to be of the same type.